services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -856,6 +861,86 @@ async def add_project_states_for_user(
     return project
 
 
+#
+#  PROJECT COMMENTS -------------------------------------------------------------------
```

Comment:
I think here we will have difference of opinions in the team.
IMO these functions  (for the moment) are not going to be used outside projects plugin and therefore I would directly put them in the protected module `_handlers_project_comments`

There is no need to expose functionality outside of this plugin when it is not going to be used

In addition we are trying to get this module smaller and less coupled ... and this makes it more difficult :-)

Reply:
I agree it doesn't belong to the projects_api.py, but it also doesnt belong to the handlers. I will create a new python submodule for that: _api_project_comments.py. So it is in the "Controller - Service - Repository" pattern. In this case: "_handlers_project_comments" - "_api_project_comments" - "db"